### PR TITLE
STORM-699: storm-jdbc should support custom insert queries.

### PR DESCRIPTION
**Pull request by Parth-Brahmbhatt**

### CODE REVIEW

**Comment by harshach** [comment]
+1

File: external/storm-jdbc/README.md [hunk 1]
```diff
@@ -38,34 +63,30 @@ hikariConfigMap.put("dataSource.password","password");
...
+method will return the columns in the order in which Jdbc connection instance's `connection.getMetaData().getColumns();` method returns them.
 
+**If you specified your own insert query to `JdbcInsertBolt` you must initialize `SimpleJdbcMapper` with explicit columnschema such that the schema has columns in the same order as your insert queries.**
+For example if your insert query is `Insert into user (user_id, user_name) values (?,?)` then your `SimpleJdbcMapper` should be initialized with the following statements:
 ```java
 List<Column> columnSchema = Lists.newArrayList(
     new Column("user_id", java.sql.Types.INTEGER),
-    new Column("user_name", java.sql.Types.VARCHAR));
-    JdbcMapper simpleJdbcMapper = new SimpleJdbcMapper(columnSchema);
+    new Column("user_name", java.sql.Types.VARCHAR),
```
Comment (HeartSaVioR) [inline external/storm-jdbc/README.md]: Codes in comments seems broken. Could you have a look?

File: external/storm-jdbc/README.md [hunk 2]
```diff
@@ -13,14 +13,39 @@ public interface JdbcMapper  extends Serializable {
 }
 ```
 
-The `getColumns()` method defines how a storm tuple maps to a list of columns representing a row in a database.
+The `getColumns()` method defines how a storm tuple maps to a list of columns representing a row in a database. 
+**The order of the returned list is important. The place holders in the supplied queries are resolved in the same order as returned list.**
+For example if the user supplied insert query is `insert into table user(user_id, user_name, create_date) values (?,?, now())` the 1st item
```
Comment (HeartSaVioR) [inline external/storm-jdbc/README.md]: Could you check query again? Seems like it's strange.

Comment (Parth-Brahmbhatt) [inline external/storm-jdbc/README.md]: I had an extra "table" in it, fixed.

File: external/storm-jdbc/README.md (hunk 1 above)
Comment (Parth-Brahmbhatt) [inline external/storm-jdbc/README.md]: missed a closing bracket and ; fixed.

**Pull request merged**